### PR TITLE
LPS-165685 Rename the column OrgLabor

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for [LPS-165685](https://issues.liferay.com/browse/LPS-165685).

Just fixed a rebase error in [PortalUpgradeProcessRegistryImpl](https://github.com/liferay-usm/liferay-portal/blob/2724a20e90011d514932f6c4a05c37fa1326fe78/portal-impl/src/com/liferay/portal/upgrade/v7_4_x/PortalUpgradeProcessRegistryImpl.java#L232-L241), since the Contact_ column changes were merged.

### CODE REVIEW

**Comment by patriciajeanperez** [comment]
ci:forward

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by melodywu** [comment]
Relevant failures are not related

**Comment by drewbrokke** [comment]
ci:reevaluate:1257401_5226

File: portal-impl/src/com/liferay/portlet/usersadmin/util/UsersAdminImpl.java [hunk 1]
```diff
@@ -738,10 +738,10 @@ public List<OrgLabor> getOrgLabors(ActionRequest actionRequest) {
 			ParamUtil.getString(actionRequest, "orgLaborsIndexes"), 0);
 
 		for (int orgLaborsIndex : orgLaborsIndexes) {
-			long typeId = ParamUtil.getLong(
+			long listTypeId = ParamUtil.getLong(
 				actionRequest, "orgLaborTypeId" + orgLaborsIndex, -1);
```
Comment (drewbrokke) [inline portal-impl/src/com/liferay/portlet/usersadmin/util/UsersAdminImpl.java:742]: Is this parameter name something we can control? Let's double-check.

`"orgLaborTypeId" -> "orgLaborListTypeId"`

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@gislayne-vitorino just the one comment about the parameter name, everything else looks good. Thanks!

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:relevant

**Comment by gislayne-vitorino** [comment]
ci:test:relevant

**Comment by patriciajeanperez** [comment]
Pushed to BChan
https://github.com/brianchandotcom/liferay-portal/pull/125827

**Pull request closed without merging**